src/Navigation/Xamarin.Forms/Navigation.Xamarin.Forms/NavigationServiceExtensions.cs

```diff
@@ -227,5 +181,20 @@ public static class NavigationServiceExtensions
 
             return registrationInfo;
         }
+
+
+
+        private static void ThrowIfNavigationServiceIsOfWrongType(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private method 'ThrowIfNavigationServiceIsOfWrongType'.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5598919)